planner/core/exhaust_physical_plans.go

```diff
@@ -387,6 +387,93 @@ func (p *LogicalJoin) constructIndexJoin(
 	return []PhysicalPlan{join}
 }
 
+func findIndexScanAndTableScan(p PhysicalPlan) (is *PhysicalIndexScan, ts *PhysicalTableScan) {
+	var children []PhysicalPlan
+	switch x := p.(type) {
+	case *PhysicalTableReader:
+		children = x.TablePlans
+	case *PhysicalIndexReader:
+		children = x.IndexPlans
+	case *PhysicalIndexLookUpReader:
+		children = append(x.IndexPlans, x.TablePlans...)
```

Comment:
This may cause `runtime.growslice()`. Considered that we only need to read the child information, I think we don't need to use a slice to collect all the child operators. We can just visit the child to improve the planner performance by reducing the gc pressure.

Reply:
OK.